play.html

```diff
@@ -10,15 +10,22 @@
 </head>
 
 <body style="justify-content: flex-end;">
-
     <span id="result"></span>
 
-    <p id="warning"></p>
 
-    <button id='clipboard' onclick="copyToClipBoard()" style="display: none">Copy your score to clipboard</button>
+    <div class="history">
 
-    <table id='guesses' class="history">
-    </table>
+        <table id='guesses'>
+        </table>
+        <p id="warning"></p>
+
+        <button id='clipboard' onclick="copyToClipBoard()" style="display: none">Copy your score to clipboard</button>
+
+        <button id="pickWord" onclick="location.href='index.html'" style='display:none'>
```

Comment:
This can just be `<a href="index.html">Pick another word</a>` instead of messing with `onclick` event and changing `location.href`

Reply:
It's not much of a difference. I prefer the button

Reply:
It's a big difference. Using `<a>` creates a proper link that you can style to look like a button if you want.

But a button IS NOT A LINK. This means that browsers can't show where the link is going and they can't preload the contents making the experience harder to understand and slower for users. Additionally this is very hard to use for the blind using screen readers, because they're not expecting buttons to navigate the page.

In general you should avoid setting location.href unless you have a very complex problem that you can't solve with a link.

Reply:
very well